.github/workflows/connector-performance-command.yml

```diff
@@ -124,7 +133,7 @@ jobs:
       - name: KIND Kubernetes Cluster Setup
         uses: helm/kind-action@v1.4.0
         with:
-          config: ./tools/bin/source-harness-kind-cluster-config.yaml
+          config: "./tools/bin/${{ steps.which-harness.outputs.harness_type }}-kind-cluster-config.yaml"
```

Comment:
nice I like this.

Reply:
🙏 
the whole what to escape, quote, put in a single ${, put in a double ${{

a never ending quandary